crates/subspace-archiving/Cargo.toml

```diff
@@ -17,6 +17,7 @@ merkletree = "0.21.0"
 parity-scale-codec = { version = "2.3.0", features = ["derive"], default-features = false }
 thiserror = "1.0.29"
 typenum = "1.14.0"
+serde = { version = "1.0.130", optional = true, features = ["derive"] }
 subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives", default-features = false }
```

Comment:
Looks like these 2 are not sorted alphabetically and `serde` in `std` feature below too